.github/workflows/gradle.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
       # Integration test
       - name: Integration test
-        run: ./gradlew :sechub-integrationtest:startIntegrationTestInstances :sechub-systemtest:integrationtest :sechub-integrationtest:integrationtest :sechub-integrationtest:stopIntegrationTestInstances -Dsechub.build.stage=all --console=plain
+        run: ./gradlew :sechub-integrationtest:startIntegrationTestInstances :sechub-integrationtest:integrationtest :sechub-systemtest:integrationtest :sechub-integrationtest:stopIntegrationTestInstances -Dsechub.build.stage=all --console=plain
```

Comment:
Please add a comment explaining, that the order is important: first the integrationtest afterwards the systemtest